integration-test-common/src/test/java/org/apache/gravitino/integration/test/util/AbstractIT.java

```diff
@@ -248,15 +248,8 @@ public static String startAndInitMySQLBackend() {
     }
   }
 
-  @ParameterizedTest
-  @CsvSource({
-    "embedded, jdbcBackend",
-    "embedded, kvBackend",
-    "deploy, jdbcBackend",
-    "deploy, kvBackend"
-  })
```

Comment:
Would removing these have no effect?

Reply:
It's pointless and won't actually happen.